workers/data_refinery_workers/downloaders/array_express.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import absolute_import, unicode_literals
+import os
+import requests
+from celery import shared_task
+from celery.utils.log import get_task_logger
+from data_refinery_models.models import Batch, DownloaderJob
+from data_refinery_workers.downloaders import utils
+
+logger = get_task_logger(__name__)
+
+# chunk_size is in bytes
+CHUNK_SIZE = 4096
+# This path is within the Docker container.
+ROOT_URI = "/home/user/data_store/"
+
+
+@shared_task
+def download_array_express(job_id):
+    job = (DownloaderJob
+           .objects
+           .filter(id=job_id)
+           [:1]
```

Comment:
I think you're very right. I think I found this way of doing it online without understanding what was going on well enough. I agree that this should use the `get` method.